README-dev.md

```diff
@@ -1,98 +1,136 @@
 # Mina
 
-Mina is a new cryptocurrency protocol with a lightweight, constant-sized blockchain.
+Mina is a cryptocurrency protocol with a lightweight, constant-sized blockchain.
 
-- [Node Developers Overview Page](https://docs.minaprotocol.com/node-developers)
-- [Repository Readme](README.md)
+- [Node Developers Overview](https://docs.minaprotocol.com/node-developers)
+- [Mina README](README)
 
-If you haven't seen it yet, [CONTRIBUTING.md](CONTRIBUTING.md) has information
-about our development process and how to contribute. If you just want to build
+For information about our development process and how to contribute, see [CONTRIBUTING.md](CONTRIBUTING.md). If you want to build
 Mina, this is the right file!
 
 ## Building Mina
 
-Building Mina can be slightly involved. There are many C library dependencies that need
-to be present in the system, as well as some OCaml-specific setup.
+Building Mina is involved because many C library dependencies must be present in the system. OCaml-specific setup is also required.
 
-If you are already a Nix user, or are comfortable installing Nix, it
-can be an easy way to build Mina locally. See
-[nix/README.md](./nix/README.md) for more information and
-instructions.
+If you are already a Nix user, or are comfortable installing Nix, you already have a way to build Mina locally. For information and
+instructions, see [nix/README.md](./nix/README.md).
 
-Currently, Mina builds/runs on Linux & macOS. MacOS may have some issues that you can track [here](https://github.com/MinaProtocol/coda/issues/962).
+Mina builds and runs on Linux and macOS.
 
-The short version:
+Quick start instructions:
 
 1.  Start with Ubuntu 18 or run it in a virtual machine
-2.  Set github repos to pull and push over ssh: `git config --global url.ssh://git@github.com/.insteadOf https://github.com/`
-    - To push branches to repos in the MinaProtocol or o1-labs organisations, you must complete this step. These repositories do not accept the password authentication used by the https URLs.
-3.  Pull in our submodules: `git submodule update --init --recursive`
-    - This might fail with `git@github.com: Permission denied (publickey).`. If that happens it means
-      you need to [set up SSH keys on your machine](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
-4.  Run `git config --local --add submodule.recurse true`
+2.  The MinaProtocol and o1-labs repositories do not accept the password authentication used by the https URLs. You must set GitHub repos to pull and push over ssh: 
+
+    ```sh
+    git config --global url.ssh://git@github.com/.insteadOf https://github.com/
+    ```
```

Comment:
@Sventimir  can you make direct updates with that suggestion? my updates did not introduce new content but just provided grooming for clarity
Since this change is out of scope for editorial grooming, we can skip your suggestion?

Reply:
Yes, I'll prepare a separate PR.